src/components/Header/Header.scss

```diff
@@ -0,0 +1,33 @@
+.logo {
+  display: flex;
+  justify-content: flex-start;
+  width: 50px;
+  height: 50px;
+  margin-right: 24px;
+}
+
+.navPages {
+  display: flex;
+  justify-content: flex-start;
+  flex-grow: 1;
+}
+
+.navPage {
+  margin: 4px;
+}
+
+.socialNetworks {
+  display: flex;
+  justify-content: flex-end;
+}
+.socialNetworks > .socialNetwork {
```

Comment:
add two classes to increase the weight of selectors (using one class the styles didn't work)